package.json

```diff
@@ -23,29 +23,30 @@
     "pull-cat": "^1.1.11",
     "pull-notify": "^0.1.1",
     "pull-stream": "^3.6.14",
-    "ssb-bendy-butt": "~0.12.5",
```

Comment:
This is now just a feed format that has to be installed as an attachment on ssb-db2, so ssb-meta-feeds doesn't import it directly.